internal/framework/runnables/cronjob.go

```diff
@@ -13,6 +13,8 @@ import (
 type CronJobConfig struct {
 	// Worker is the function that will be run for every cronjob iteration.
 	Worker func(context.Context)
+	// ReadyCh represents if the cronjob is ready to start.
```

Comment:
nit:  improving the description
```suggestion
	// ReadyCh delays the start of the job until the channel is closed.
```

Reply:
so much better